### PR TITLE
Remove name and description and cleanup

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java [hunk 1]
```diff
@@ -149,42 +143,41 @@ public void testStudyRepository() {
...
-        assertTrue(studyRepository.findByUserIdAndStudyName("foo", "mystudy").isPresent());
         assertEquals(1, studyRepository.findAllByUserId("foo").size());
 
         // updates
         savedStudyEntity1.setLoadFlowResult(loadFlowResultEntity);
         savedStudyEntity1.setLoadFlowParameters(loadFlowParametersEntity);
         studyRepository.save(savedStudyEntity1);
 
-        StudyEntity savedStudyEntity1Updated = studyRepository.findByUserIdAndStudyName("foo", "mystudy").get();
+        //StudyEntity savedStudyEntity1Updated = studyRepository.findByUserIdAndStudyName("foo", "mystudy").get();
```
Comment (niconoir) [inline src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java:156]: Remove commented line

File: src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java [hunk 2]
```diff
@@ -149,42 +143,41 @@ public void testStudyRepository() {
...
+        StudyEntity savedStudyEntity1Updated = studyRepository.findById(studyEntity1.getId()).get();
         assertNotNull(savedStudyEntity1Updated.getLoadFlowResult());
         assertEquals(2, savedStudyEntity1Updated.getLoadFlowResult().getComponentResults().size());
         assertNotNull(savedStudyEntity1Updated.getLoadFlowParameters());
 
         savedStudyEntity1Updated.setLoadFlowStatus(LoadFlowStatus.CONVERGED);
         studyRepository.save(savedStudyEntity1Updated);
-        savedStudyEntity1Updated = studyRepository.findByUserIdAndStudyName("foo", "mystudy").orElse(null);
+        //savedStudyEntity1Updated = studyRepository.findByUserIdAndStudyName("foo", "mystudy").orElse(null);
```
Comment (niconoir) [inline src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java:164]: Remove commented line

**Review by niconoir — COMMENTED** [review]
Just 2 commented lines to remove

File: src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java]: Done

File: src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/study/server/repository/RepositoriesTest.java]: Done

**Review by niconoir — APPROVED** [review]

**Pull request merged**